braindecode/augmentation/functional.py

```diff
@@ -1041,3 +1041,54 @@ def segmentation_reconstruction(
         return aug_data, aug_label
 
     return aug_data, y
+
+
+def mask_encoding(X, y, time_start, segment_length, splits):
+    """Replaces a contiguous part (or parts) of all channels by zeros
+    (if more than one segment, it may overlap).
+
+     Implementation based on [1]_
+
+     Parameters
+     ----------
+     X : torch.Tensor
+         EEG input example or batch.
+     y : torch.Tensor
+         EEG labels for the example or batch.
+     time_start : torch.Tensor
+         Tensor of integers containing the position (in last dimension) where to
+         start masking the signal. Should have "splits" times the size of the first
+         dimension of X (i.e. "splits" start positions per example in the batch).
+     segment_length: int
+         Length of each segment to zero out.
+     splits : int
```

Comment:
```suggestion
     n_segments : int
```
as you use the word segment elsewhere.